### PR TITLE
NONE: Document existing features

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ Read-only queries can be sent to the replica, while others have to go to the mai
 The `db-replica` API automatically routes the queries to the correct node.
 It integrates at the `java.sql.Connection` level, so you don't have to hunt down hundreds of queries manually.
 
+## Features
+
+- Automatic switching between main and replica databases (`com.atlassian.db.replica.api.DualConnection`).
+- Configurable consistency model (`com.atlassian.db.replica.spi.ReplicaConsistency`).
+- Configurable circuit breaker (`com.atlassian.db.replica.spi.circuitbreaker.CircuitBreaker`).
+- Configurable main/replica split instrumentation (`com.atlassian.db.replica.spi.DualCall`).
+- Connection state change listener (`com.atlassian.db.replica.spi.state.StateListener`).
```
Comment (dagguh) [inline README.md:17]: Very nice touch 👌 
It reduces the barrier of entry quite a lot.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,14 @@ Read-only queries can be sent to the replica, while others have to go to the mai
 The `db-replica` API automatically routes the queries to the correct node.
 It integrates at the `java.sql.Connection` level, so you don't have to hunt down hundreds of queries manually.
 
+## Features
+
+- Automatic switching between main and replica databases (`com.atlassian.db.replica.api.DualConnection`).
```
Comment (dagguh) [inline README.md:13]: Wanna link to the source file?

**Review by dagguh — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (wyrzyk) [reply README.md:13]: Yes. But I always forget the syntax.

Comment (dagguh) [reply README.md:13]: Generally: `[label](address)`
Specifically: ``[`com.atlassian.db.replica.api.DualConnection`](src/main/java/com/atlassian/db/replica/api/DualConnection.java)``

**Review by dagguh — APPROVED** [review]
Links work for me 👍

**Pull request merged**